src/ray/core_worker/reference_count.cc

```diff
@@ -27,59 +27,55 @@ void ReferenceCounter::AddOwnedObject(const ObjectID &object_id, const TaskID &o
 
 void ReferenceCounter::AddLocalReference(const ObjectID &object_id) {
   absl::MutexLock lock(&mutex_);
-  auto entry = object_id_refs_.find(object_id);
-  if (entry == object_id_refs_.end()) {
+  auto it = object_id_refs_.find(object_id);
+  if (it == object_id_refs_.end()) {
     // TODO: Once ref counting is implemented, we should always know how the
```

Comment:
I think this TODO is no longer relevant since it's possible to add the first local reference before ownership info is added. Can you remove it and change it to a note that the ownership info must be added later in AddBorrowedObject?